test/Thesis/SqlGeneratorSpec.hs

```diff
@@ -0,0 +1,83 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Thesis.SqlGeneratorSpec (spec) where
+
+import Data.Text (Text, append)
+import Database.PostgreSQL.Simple.Types (Identifier(..))
+import Test.Hspec
+import Test.Hspec.QuickCheck
+import Test.QuickCheck.Instances.Text()
+
+import Thesis.Ast
+import Thesis.Query
+import Thesis.SqlGenerator.Internal
+import Thesis.SqlBuilder (SqlPart(..), Parameter(..))
+import Thesis.ValueGuard
+
+testExpr :: Expr
+testExpr = (Literal (Value ("test" :: String)))
+
+testParameter :: Parameter
+testParameter = Parameter ("test" :: String)
+
+spec :: Spec
+spec =
+  describe "Thesis.SqlGenerator" $ do
```

Comment:
We can skip this describe, the testing framework automatically wraps everything in a describe with the module's name.